### DISCUSSION TITLE
Prevent countries and towns from being created in designated areas.

### DISCUSSION THREAD

**Discussion by mtmtyu** [TownyAdvanced/Towny#7066] [Q&A] [answered]
Is there a way to prevent a specified chunk from creating a country or town, or from acquiring land?

**Comment by LlmDl** [comment, marked as answer]
Either claim it yourself with an admin town, (see the Mayor section of the HowTownyWorks wikipage,) or use WorldGuard-Towny from the [website](https://townyadvanced.github.io).
